docs/pages/release_notes.md

```diff
@@ -14,6 +14,16 @@ This is a {{ site.pmd.release_type }} release.
 
 ### 🚀 New and noteworthy
 
+#### Collections exposed as XPath attributes
+
+Up to now, all AST node getters would be exposed to XPath, as long as the return type was a primitive (boxed or unboxed), String or Enum. That meant that collections, even of these basic types, were not exposed, so for instance accessing Apex's `ASTUserClass.getInterfaceNames()` to list the interfaces implemented by a class was impossible from XPath, and would require writing a Java rule to check it.
```

Comment:
The page writing_xpath_rules.md probably needs to be updated as well.

Reply:
Done